openscap.yaml

```diff
@@ -99,6 +99,7 @@ subpackages:
               import openscap_py
               import oscap_docker_python
         - runs: |
+            ln -s /usr/bin/python3.13 /usr/bin/python3
```

Comment:
This only causes the test to pass and the oscap-docker script is not usable if you just install openscap-docker.
I've fixed this in https://github.com/wolfi-dev/os/pull/49765